e2e/capacity/transactions.test.ts

```diff
@@ -199,12 +199,15 @@ describe('Capacity Transactions', function () {
         before(async function () {
           capacityKeys = createKeys('CapacityKeys');
           capacityProvider = await createMsaAndProvider(fundingSource, capacityKeys, 'CapacityProvider', FUNDS_AMOUNT);
+          const numberOfTests = BigInt(this.test!.parent!.tests.length);
+          // Stake the amount for each test
+          await assert.doesNotReject(
+            stakeToProvider(fundingSource, fundingSource, capacityProvider, numberOfTests * amountStaked)
+          );
```

Comment:
Instead of staking enough for each call, we instead just stake in the `before` based on how many tests there are